src/hooks/use-handle-snapshot.ts

```diff
@@ -28,13 +28,11 @@ export const useHandleSnapshot = (storeExists = true) => {
     const resp = await saveSnapshot(docToJSON())
     if ('data' in resp) {
       execCmd(trackCommands.applyAndRemoveChanges())
-      setTimeout(() => {
```

Comment:
Can you add the code you supplied in the comments to be used here?

Reply:
Sure, done.